generator/src/main/java/com/scottlogic/deg/generator/generation/FieldSpecValueGenerator.java

```diff
@@ -10,11 +10,14 @@
 import com.scottlogic.deg.generator.generation.field_value_sources.FieldValueSource;
 import com.scottlogic.deg.generator.utils.JavaUtilRandomNumberGenerator;
 
+import java.time.LocalDateTime;
 import java.util.List;
 import java.util.stream.Stream;
 import java.util.stream.StreamSupport;
 
 public class FieldSpecValueGenerator {
+    private static final JavaUtilRandomNumberGenerator randomNumberGenerator = new JavaUtilRandomNumberGenerator(LocalDateTime.now().getNano());
```

Comment:
I'm not convinced that the FieldSpecValueGenerator should be responsible for newing up a random number generator. Why do we not do this in the generateRandomValues method?

Reply:
Agreed this probably should be passed in as a dependency rather than created internally